site/payment-gateway/integration.md

```diff
@@ -59,30 +59,28 @@ Here is a very simple example snippet that demonstrates this:
 ... any other content you have on your page
 
 <!-- Add the frontend payment script -->
-<script src="https://payment-gateway.holidayextras.co.uk/payment.1.0.0.min.js"></script>
+<script src="https://payment-gateway.holidayextras.co.uk/payment.2.0.0.min.js"></script>
```

Comment:
✅ This... mostly... worked for me - the postMessage never appears but I think I can pin that on it deliberately specifying an environment-specific origin - the staging script won't reach a local `gabble.holidayextras.co.uk:5000`. But it is looking up the provided email.